oauth2_http/java/com/google/auth/http/AuthHttpConstants.java

```diff
@@ -0,0 +1,18 @@
+package com.google.auth.http;
+
+import java.util.regex.Pattern;
+
+/**
+ * Constants used for auth in http
+ */
+public class AuthHttpConstants {
```

Comment:
These variables are kind of implementation related and should probably package-private. I'd recommend putting these in OAuth2Utils instead.